static/js/apps/explore/result_header_section.tsx

```diff
@@ -57,11 +58,16 @@ export function ResultHeaderSection(
         {getPlaceHeader()}
         {topicNameStr && <span> • {topicNameStr}</span>}
       </div>
-      {!_.isEmpty(props.pageMetadata.mainTopics) && !_.isEmpty(topicList) && (
-        <div className="explore-topics-box">
-          <ItemList items={topicList} showRelevantTopicLabel={true}></ItemList>
-        </div>
-      )}
+      {!_.isEmpty(props.pageMetadata.mainTopics) &&
+        !_.isEmpty(topicList) &&
+        !showFollowUpQuestions && (
```

Comment:
we should pass this through props instead of calling it like this because showFollowUpQuestions uses math.random and I believe constants from a module are initialized when the module is imported, so the number we get from math.random in this file might be different than the one used in the SuccessResult component

So, maybe add `showRelatedTopics` to `ResultHeaderSectionPropType` and use that property here instead